cmd/common/util.go

```diff
@@ -0,0 +1,58 @@
+package common
+
+import (
+	"os/user"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+func RelativeToAbsolutePath(rpath string) (string, error) {
+	if rpath == "" {
+		return rpath, nil
+	}
+
+	absolutePath := ""
+
+	// case ./ ../
+	if strings.Contains(rpath, "./") {
```

Comment:
strings package has `HasPrefix` function

you can use `string.HasPrefix(rpath,"./") | string.HasPrefix(rpath,"../")`

it can prevent path like `..../a/b/c` or like `/a/b/./c`